test_inputs/GNUmakefile

```diff
@@ -0,0 +1,21 @@
+TARGET_DIR = $(shell readlink -f $(shell pwd)/../target/test_inputs)
+BINS =	${TARGET_DIR}/hello_world/hello_world1 \
+	${TARGET_DIR}/hello_world/hello_world2 \
+	${TARGET_DIR}/fannkuch_redux/fannkuch_redux
+
+all: ${BINS}
+
+.SUFFIXES:
+.SUFFIXES: .ll .s
+
+${TARGET_DIR}/%.s: %.ll
+	mkdir -p `dirname $@`
+	llc-6.0 -relocation-model=pic -o $@ $<
+
+${TARGET_DIR}/%: ${TARGET_DIR}/%.s
+	clang-6.0 ${CFLAGS} -o $@ $< ${LDFLAGS}
+
+# Also since the binaries are put in the Cargo target dir, `cargo clean` will
```

Comment:
Comment doesn't make sense.

Reply:
No? Basically, because we output the binaries under `../target` it means that `cargo clean` will also delete them.

Reply:
"Also" what?

Reply:
Also there is another way to clean these? I'm not sure what you mean Laurie.

Reply:
"Also" doesn't make sense as the beginning of a comment. "Also since the binaries" -- also relative to what? It feels like a detective novel at the moment ;) I actually suspect the comment is best deleted; but if it's kept, it needs rephrasing.

Reply:
Right, i see what you mean now :P 

Yes, we can probably kill this, but I thought you would ask why I'm outputting binaries in such a strange place.